samtranslator/model/eventsources/push.py

```diff
@@ -218,6 +220,15 @@ def to_cloudformation(self, **kwargs):
             )
             resources.extend(dlq_resources)
 
+        if self.State and self.Enabled is not None:
```

Comment:
Quick question, is there a particular reason why we also have the option of providing `Enabled: boolean` instead of just passing in `State: string`?

Reply:
Consistency with other event sources: https://github.com/aws/serverless-application-model/blob/eda646d88a8c8499c5ac2630a253e2e62d1b62be/samtranslator/model/stepfunctions/events.py#L85-L86